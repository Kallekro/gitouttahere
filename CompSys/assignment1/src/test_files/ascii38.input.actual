test_files/ascii38.input: ASCII text
