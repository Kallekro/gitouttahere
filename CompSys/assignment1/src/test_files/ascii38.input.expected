test_files/ascii38.input: Sendmail frozen configuration 
